app/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,7 @@
 <% when 'candidate_interface' %>
   <%= render(HeaderComponent.new(classes: "app-header--#{HostingEnvironment.environment_name}",
                                  service_name: service_name,
-                                 service_url: service_link,
+                                 service_url: govuk_apply_start_page,
```

Comment:
Header should link to dashboard/application page when signed in, but not be linked at all when signed out. Not sure this achieves that?

Reply:
🤔 So a few things to consider here.

1. I think this needs to revert back to the `service_url` - this url dynamically changes based on the path ('/provider'. '/support' etc) and redirects back to their respective starting (home?) pages
2. Given the `service_url` is based on `path`, this then may present a bigger technical problem if we want to then also change the url based on whether the user is signed in or not. This requires more logic and code changes.
3. I was looking at Find and the header does not link back to the GOVUK start page. Is this definitely the behaviour we want?
4. Longing for that 'one line code change' mvp mentioned in the ticket! 😆

Reply:
1. Yeah, that sounds best. Just need to work out where `service_url` goes when signed out.
2. Desirable, but not necessary, so ignore for now.
3. Ideally, but not important.
4. 😭